app/components/chat-text-options.js

```diff
@@ -0,0 +1,16 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+export default class ColorTextSelector extends Component {
+  @tracked style = "color: yellow"
```

Comment:
Do we care about semicolons? Not sure if there's anything in the linting settings about it...

Reply:
Updated this component to use semicolons and use the same method of datafruits-chat.js of updating the elements style (@oneWay). I was wondering how I can move the chat-text-options.hbs to the templates directory? Probably should have used the cli?

Reply:
Oh, I think in newer versions of ember, templates are in the same directory as components, so maybe we should start switching to that structure..